src/Selenium2Library/keywords/_element.py

```diff
@@ -1,14 +1,32 @@
 from selenium.webdriver.common.keys import Keys
 from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.remote.webelement import WebElement
 from Selenium2Library import utils
 from Selenium2Library.locators import ElementFinder
 from keywordgroup import KeywordGroup
 
+try:
+    basestring  # attempt to evaluate basestring
+    def isstr(s):
+        return isinstance(s, basestring)
+except NameError:
+    def isstr(s):
+        return isinstance(s, str)
+
 class _ElementKeywords(KeywordGroup):
 
     def __init__(self):
         self._element_finder = ElementFinder()
 
+    # Public, get element(s)
+
+    def get_webelements(self, locator):
```

Comment:
How about ```Find Element``` and ```Find Elements``` then? It's not necessarily as clear, but that matches up with the selenium API. (I'm not really necessarily opposed to ```Get WebElements``` though).